test/test_contrib.py

```diff
@@ -276,3 +276,31 @@ def test_gradcheck(self, device, dtype):
             torch.arange(16., device=device, dtype=dtype).view(1, 1, 4, 4), window_size=(2, 2), stride=(2, 2))
         input = utils.tensor_to_gradcheck_var(patches)  # to var
         assert gradcheck(kornia.contrib.combine_tensor_patches, (input, (2, 2), (2, 2)), raise_exception=True)
+
+
+class TestLambdaModule:
+
+    def add_2_layer(self, tensor):
+        return tensor + 2
+
+    def test_smoke(self, device, dtype):
+        B, C, H, W = 1, 3, 4, 5
+        input = torch.rand(B, C, H, W, device=device, dtype=dtype)
+        func = self.add_2_layer
+        if not callable(func):
+            raise TypeError(f"Argument lambd should be callable, got {repr(type(func).__name__)}")
+        assert isinstance(kornia.contrib.Lambda(func)(input), torch.Tensor)
+
+    def test_lambda(self, device, dtype):
```

Comment:
Could you please add the test case for having arguments and keyword arguments?

Reply:
Should I make the input variable as the parameter? and then pass the 'B C H W' as the parameters? Or are you trying to say something else?